client/src/components/atoms/Skeletons/BoxSkeleton/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { skeletonAnimation } from "~/utils/skeletonAnimation";
+
+const BoxSkeleton = ({ className }: { className?: string }) => {
```

Comment:
I thought this was templates folder

`templates/skeleton`
 / BoxSkeleton
 / ImageSkeleton
 / LineSkeleton

Reply:
no sir because a template has children. and we use skeletons as a child on our page.